test/rules/label-required/valid-boundary-event.bpmn

```diff
@@ -2,7 +2,7 @@
 <bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_027stl5" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="1.16.2">
   <bpmn:process id="Process_1" isExecutable="true">
     <bpmn:subProcess id="SubProcess_0w2vmlo" />
-    <bpmn:boundaryEvent id="BoundaryEvent_1wvvk9q" attachedToRef="SubProcess_0w2vmlo" />
+    <bpmn:boundaryEvent id="BoundaryEvent_1wvvk9q" name="Q" attachedToRef="SubProcess_0w2vmlo" />
```

Comment:
We can simply remove the boundary event since the only valid case is a boundary event with name.